ui/component/preorderAndPurchaseContentCard/view.jsx

```diff
@@ -146,6 +164,8 @@ export default function PreorderContent(props: Props) {
 
     setWaitingForBackend(true);
 
```

Comment:
Is it missing it's `false` counterpart, or perhaps handled indirectly somewhere?

Reply:
Should be unmounted/hidden